src/redux/categories/categories.js

```diff
@@ -0,0 +1,13 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] according to the requirements The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux).